### DISCUSSION TITLE
atom feed

### DISCUSSION THREAD

**Discussion by Dieterbe** [hugo-sid/hugo-blog-awesome#224] [Feature requests]
apparently i have some users that are still requesting /index.atom, which doesn't work anymore when i switched to this theme. i'm not 100% sure it's the theme's responsibility, but i couldn't find any docs related to atom on the hugo website.

**Comment by pitscher** [comment]
Hey @Dieterbe, I'm curious: What is the purpose of `index.atom`?
Took 2 mins to ask Google and got even more confused 😅

**Reply by Dieterbe** [reply]
Atom is a feed format, an alternative to RSS
